include/z64item.h

```diff
@@ -308,6 +308,32 @@ typedef enum ItemId {
     /* 0xA1 */ ITEM_GOLD_DUST_2,
     /* 0xA2 */ ITEM_HYLIAN_LOACH_2,
     /* 0xA3 */ ITEM_SEAHORSE_CAUGHT,
+    // Regions used with item pause menu
+    /* 0xA4 */ ITEM_REGION_GREAT_BAY,
+    /* 0xA5 */ ITEM_REGION_ZORA_HALL,
+    /* 0xA6 */ ITEM_REGION_ROMANI_RANCH,
+    /* 0xA7 */ ITEM_REGION_DEKU_PALACE,
+    /* 0xA8 */ ITEM_REGION_WOODFALL,
+    /* 0xA9 */ ITEM_REGION_CLOCK_TOWN,
+    /* 0xAA */ ITEM_REGION_SNOWHEAD,
+    /* 0xAB */ ITEM_REGION_IKANA_GRAVEYARD,
+    /* 0xAC */ ITEM_REGION_IKANA_CANYON,
+    /* 0xAD */ ITEM_REGION_GORON_VILLAGE,
+    /* 0xAE */ ITEM_REGION_STONE_TOWER,
+    // Owl Warps used with item pause menu
+    // Overlapping Owl Warps with Regions:
+    /* 0xA8 */ ITEM_OWL_WARP_WOODFALL = ITEM_REGION_WOODFALL,
```

Comment:
Oh, what a weird way of handling regions

Reply:
I feel like these should all be just `ITEM_REGION_*` because I just don't see the justification for the `ITEM_OWL_WARP` enums given that they have little use and could be confused for OwlWarpId enum values.

Reply:
There's an important distinction between `REGION` and `OWL_WARP` when it comes to warping and the world map. It might be better to have a common name between them and treat it as 1 enum perhaps? Something like `ITEM_MAP_AREA_XXX` or `ITEM_MAP_POINT_XXX`, personally I think `ITEM_MAP_POINT_XXX` or `ITEM_MAP_POINT_NAME_XXX` might work since these entries in `ItemId` are designed specifically for the map name and are named `gMapPointNameXXX`?

Or maybe just drop the `Name` so:
`gMapPointNameGreatBayENGTex` -> `gMapPointGreatBayENGTex`
`ITEM_REGION_GREAT_BAY` -> `ITEM_MAP_POINT_GREAT_BAY`
`ITEM_OWL_WARP_GREAT_BAY_COAST` -> `ITEM_OWL_WARP_GREAT_BAY_COAST`?

I think it's still important to comment/show that there's overlapping/reuse of map points between regions and owl warps though

Reply:
I think I prefer the `ITEM_MAP_POINT_XXX` naming scheme

Reply:
I do like `ITEM_MAP_POINT_NAME`

> I think it's still important to comment/show that there's overlapping/reuse of map points between regions and owl warps though

I don't see it being necessary to do so in the `ITEM_` enum if the values are purely for fetching the location name texture since there is no reason to treat them differently. I feel the intent is that there's simply a list of location names that both world and owl maps can grab from. To me, the only oddity/optimization that needs documenting is the coupling of the `ITEM_` enum order with the `RegionId`/MapNameStatic.xml order; kaleido can handle the rest.

Reply:
There does need to be a few things synced for this system to work. The entries of the `ITEM_MAP_POINT_` needs to align with `map_name_static`, and the first entries of `map_name_static` also needs to align with `RegionId`, meaning the first entries of `ITEM_MAP_POINT_` does need to be in the same order as `RegionId`. I've added comments trying to explain this, having only `ITEM_MAP_POINT_` does lose a bit of that safeguard and just relies on comments, but other ideas are welcome